src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const a = document.querySelectorAll('a');
+const largeImg = document.querySelector('#largeImg');
+
+for (const item of a) {
+  item.addEventListener('click', (evnt) => {
+    evnt.preventDefault();
+    largeImg.src = item.href;
```

Comment:
Ensure that the anchor elements have valid `href` attributes pointing to image URLs. If `item.href` does not contain a valid image URL, the `largeImg.src` assignment will not work as intended. Consider adding error handling or validation to ensure the `href` is a valid image URL.